### PR TITLE
Add php 7.4 to docker image

### PR DESCRIPTION
**Pull request by TeBoring**

### CODE REVIEW

File: kokoro/linux/dockerfile/test/php/Dockerfile [hunk 1]
```diff
@@ -232,6 +235,48 @@ RUN wget -O phpunit https://phar.phpunit.de/phpunit-7.phar \
   && cp phpunit /usr/local/php-7.3/bin \
   && mv phpunit /usr/local/php-7.3-zts/bin
 
+# php 7.4
+RUN wget http://ftp.gnu.org/gnu/bison/bison-3.0.1.tar.gz -O /var/local/bison-3.0.1.tar.gz
```
Comment (acozzette) [inline kokoro/linux/dockerfile/test/php/Dockerfile]: Could you make this an https link just to be safe?

File: kokoro/linux/dockerfile/test/php_32bit/Dockerfile [hunk 2]
```diff
@@ -218,6 +221,46 @@ RUN wget -O phpunit https://phar.phpunit.de/phpunit-7.phar \
   && cp phpunit /usr/local/php-7.3/bin \
   && mv phpunit /usr/local/php-7.3-zts/bin
 
+# php 7.4
+RUN wget http://ftp.gnu.org/gnu/bison/bison-3.0.1.tar.gz -O /var/local/bison-3.0.1.tar.gz
```
Comment (acozzette) [inline kokoro/linux/dockerfile/test/php_32bit/Dockerfile]: Same with this link

File: kokoro/linux/dockerfile/test/php/Dockerfile (hunk 1 above)
Comment (TeBoring) [reply kokoro/linux/dockerfile/test/php/Dockerfile]: Done

File: kokoro/linux/dockerfile/test/php_32bit/Dockerfile (hunk 2 above)
Comment (TeBoring) [reply kokoro/linux/dockerfile/test/php_32bit/Dockerfile]: done

File: kokoro/linux/dockerfile/test/php/Dockerfile (hunk 1 above)
Comment (TeBoring) [reply kokoro/linux/dockerfile/test/php/Dockerfile]: done

**Comment by acozzette** [comment]
LGTM

**Pull request merged**